content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,149 @@
-  also called environment, which is a representation of an organization, to manage and register apps.
-| Ultimately, `Register an Application <https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_,
-  choosing the appropriate supported account type.
+To sync the Outlook Calendar with Odoo's Calendar, a Microsoft Azure account is needed. Creating an
+account is free for users who have never tried or paid for Azure. For more information, `click here
+<https://azure.microsoft.com/en-us/free/?WT.mc_id=A261C142F>`_.
 
-.. note::
-   When configuring your platform settings, choose *Web*.
+Refer to `Microsoft's documentation
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant>`_
+on how to set up an Azure AD Tenant (also called an *environment*), which is a representation of an
+organization to manage and register apps.
+
+Then, `Register an Application
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_,
+choosing the appropriate supported account type. Users who wish to connect their Outlook calendar
+to Odoo should select the :guilabel:`Accounts in any organizational directory (Any Azure AD
+directory - Multitenant and personal Microsoft accounts (e.g. Skype, Xbox)` option for
+:guilabel:`Supported account types`.
 
-   .. image:: outlook_calendar/platform_configurations.png
-      :align: center
-      :alt: Adding a platform in Microsoft Azure for Outlook calendar sync in Odoo
+
+When configuring the platform settings, choose :guilabel:`Web`.
+
+.. image:: outlook_calendar/platform_configurations.png
+   :align: center
+   :alt: Adding a platform in Microsoft Azure for Outlook calendar sync in Odoo.
 
 .. important::
-   | For the **Redirect URI restrictions**, copy your Odoo database URI followed by
-     ``/microsoft_account/authentication``. Example:
+   For the :guilabel:`Redirect URI restrictions`, copy the Odoo database URI (URL) followed by
+   `/microsoft_account/authentication`. For example, 
+   `(https://www.companyname.odoo.com/microsoft_account/authentication)`.
+
+   .. image:: outlook_calendar/redirect_URIs.png
+      :align: center
+      :alt: Odoo's database URI that is accepted when Microsoft returns authentication.
 
-      .. image:: outlook_calendar/redirect_URIs.png
-         :align: center
-         :alt: Odoo's database URI that is accepted when microsoft returns authentication
+   For more information on the restrictions and limitations of URIs, `check this page <https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
 
-   | For more information on the restrictions and limitations of URIs, `check this page <https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
+Regarding credentials, the user **must** :guilabel:`Add a client secret`, which allows Odoo to
```

Comment:
Would italicize here for emphasis, as the bold formatting looks like it merges with the guilabel.
```suggestion
Regarding credentials, the user *must* :guilabel:`Add a client secret`, which allows Odoo to
```